bk-nodes/bk_nodes.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+# ### mysqlclient ###
+# pip install mysqlclient
+# https://github.com/PyMySQL/mysqlclient-python
+# https://mysqlclient.readthedocs.io/
+
+
+import os
+import sys
+
+
+from flask import Flask
+from flask.views import MethodView
+from flask import jsonify
+from flask.wrappers import Response
+
+from error_response import ErrorResponse
+
+
+
+from flask import request
+from flask import abort, jsonify
+
+from http import HTTPStatus
+
+import config
+import datetime
+import time
+
+
+
+import bk_db
+from  bk_db import BeekeeperDB, table_fields , table_fields_index
+
+
+
+
+
+# /
+class Root(MethodView):
+    def get(self):
+        return "SAGE Beekeeper"
+
+# /log
+class Log(MethodView):
+
+
+    # example: curl -X POST -d '[{"node_id": "123", "operation":"insert", "field_name": "name", "field_value": "Rumpelstilzchen"}]' localhost:5000/log
+    
+    def post(self):
+        
+        
+        try:
+#request.get
+            postData = request.get_json(force=True, silent=False)
+       
+        except Exception as e:
+            
+            raise ErrorResponse(f"Error parsing json: { sys.exc_info()[0] }  {e}" , status_code=HTTPStatus.INTERNAL_SERVER_ERROR)
+        
+        if not postData:
+            raise ErrorResponse(f"Could not parse json." , status_code=HTTPStatus.INTERNAL_SERVER_ERROR)
+
+        
+
+        if not isinstance( postData, list ):
+            raise ErrorResponse("array expected", status_code=HTTPStatus.INTERNAL_SERVER_ERROR)
+
+        try:
+            bee_db = BeekeeperDB()
+
+            logData = []
+            default_effective_time = datetime.datetime.now(datetime.timezone.utc) # this way all operations in this submission have the exact same time
+            for op in postData:
+                
+                for f in ["node_id", "operation", "field_name", "field_value", "source"]:
+                    if f not in op:
```

Comment:
is it okay for `op` to have an extra field that is *not* in `["node_id", "operation", "field_name", "field_value", "source"]`? 

If I sent the following fields this test would still pass.  I this what you want?

```
["EXTRA", "node_id", "operation", "field_name", "field_value", "source"]
```

Reply:
ahh.. you did this cause of the optional parameter of `effective_time`.

My question is still valid about other fields that we would ignore.

Reply:
Other fields in op will simply be ignored.